packages/store/src/internal/state-factory.ts

```diff
@@ -231,6 +232,12 @@ export class StateFactory {
   private addRuntimeInfoToMeta(meta: MetaDataModel, depth: string): void {
     meta.path = depth;
     meta.selectFromAppState = propGetter(depth.split('.'), this._config);
+    const globalSelectorOptions: InternalSelectorOptions = (<any>this._config)
```

Comment:
Why do you need to cast to `any`?

Reply:
Because this is a hidden internal field and not part of the public interface